cmd/renterd/node.go

```diff
@@ -470,6 +470,10 @@ func (n *node) Shutdown() error {
 func buildStoreConfig(am alerts.Alerter, cfg config.Config, pk types.PrivateKey, logger *zap.Logger) (stores.Config, error) {
 	partialSlabDir := filepath.Join(cfg.Directory, "partial_slabs")
 
+	if cfg.Log.Database.SlowThreshold == 0 {
```

Comment:
Since this may cause existing configs to crash should it reset the default instead?

Reply:
A config with `SlowThreshold == 0` already crashes. This PR just improves the error message to tell the user which field is causing the crash rather than failing further down the stack with something more obscure.